src/NuGet.Core/NuGet.Commands/RestoreCommand/ProjectRestoreCommand.cs

```diff
@@ -116,11 +118,25 @@ public ProjectRestoreCommand(ProjectRestoreRequest request)
 
                 var runtimeGraphs = new List<RestoreTargetGraph>();
                 var runtimeTasks = new List<Task<RestoreTargetGraph[]>>();
-
+                var projectProvidedRuntimeIdentifierGraphs = new SortedList<string, RuntimeGraph>();
                 foreach (var graph in graphs)
                 {
-                    // Get the runtime graph for this specific tfm graph
-                    var runtimeGraph = GetRuntimeGraph(graph, localRepositories);
+                    // PCL Projects with Supports have a runtime graph but no matching framework.
+                    var runtimeGraphPath = _request.Project.TargetFrameworks.
+                            FirstOrDefault(e => NuGetFramework.Comparer.Equals(e.FrameworkName, graph.Framework))?
```

Comment:
I don't like that the `?` is on a different line to the `.`. To me `?.` is a single operator and should be kept together for clarity.